blockly-core/core/ui/block_space/pan_drag_handler.js

```diff
@@ -59,6 +59,19 @@ Blockly.PanDragHandler = function (blockSpace) {
    * @type {BindData}
    * @private
    */
+  this.wheelKey_ = null;
```

Comment:
That makes sense, how about s/`Key`/`EventKey` for the usages? Those would become e.g. `wheelEventKey_`, `mousewheelEventKey_`